src/classes/GS_AdminSetup_TEST.cls

```diff
@@ -0,0 +1,91 @@
+/*
+    Copyright (c) 2020, Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group Getting Started
+* @description Class to test GS_AdminSetup class.
+*/
+@isTest
+private with sharing class GS_AdminSetup_TEST {
+  
+    /**
+    *  @description This test data if wraped in GS_AdminSetup.getChecklist are the same in the metadata
+    */
+    @isTest
+    static void getChecklistsTest() {
+        List<GS_AdminSetup.ChecklistSection> checklists = GS_AdminSetup.getChecklists();
+        List<GS_Admin_Checklist_Section__mdt> metadataSections = [ 
```

Comment:
In a unit test this query will always return an empty set (because it's not possible to insert test data into a CMDT object). Does the unit test account for that, or is this test always skipped when the CMDT object is empty? If it's the latter (which I suspect is happening), it may be necessary to implement mocking methods to handle dynamically mocking the CMDT data for these tests. @dcaballeroSf

Reply:
The test sees the CMDT records(https://developer.salesforce.com/forums/?id=9060G0000005qUEQAY). Also yesterday I debugged this test due to other work item i was working on and I saw that actually it sees all the records of the CMDT (even when seeAllData=false). So in my opinion, no mocking is needed

Reply:
Ah. I forgot that we're including the CMDT data in the package so the objects will be populated even in our automated build environments. Thanks for clarifying.